oml/registry/losses.py

```diff
@@ -10,15 +12,20 @@
     "triplet": TripletLoss,
     "triplet_plain": TripletLossPlain,
     "triplet_with_miner": TripletLossWithMiner,
+    "arcface": ArcFaceLoss,
+    "mlp_arcface": ArcFaceLossWithMLP,
 }
 
 
 def get_criterion(name: str, **kwargs: Dict[str, Any]) -> nn.Module:
+    constructor = LOSSES_REGISTRY[name]
     if "miner" in kwargs:
         miner = get_miner_by_cfg(kwargs.pop("miner"))
-        return LOSSES_REGISTRY[name](miner=miner, **kwargs)
+        kwargs = {k: v for k, v in kwargs.items() if k in inspect.signature(constructor).parameters}
```

Comment:
let's move the duplicated code to `oml.utils.misc`:
```
def remove_unused_kargs(kwargs, constructor):
    pass
```
we will also reuse this code later for samplers